pint-abi-gen-tests/test-pkgs/counter/src/contract.pnt

```diff
@@ -2,13 +2,8 @@ storage {
     counter: int,
 }
 
-predicate Init {
-    var value: int;
-    state counter: int = storage::counter;
-    constraint counter' == value;
-}
-
 predicate Increment {
     state counter: int = storage::counter;
-    constraint counter' == counter + 1;
+    constraint (counter == nil && counter' == 0)
+        || counter' == counter + 1;
```

Comment:
Formatting feels a little weird but might just be worth waiting till pintfmt is fixed to change this

Reply:
`pintfmt` isn't getting a proper update any time soon.

Reply:
I'll put it all on one line instead